nativelib/src/main/scala/scala/scalanative/unsafe/package.scala

```diff
@@ -99,6 +99,10 @@ package object unsafe {
   /** C-style alignment operator. */
   @alwaysinline def alignmentof[T](implicit tag: Tag[T]): CSize = tag.alignment
 
+  // Scala 3 does not allow to use extern method and extern annotation class
+  // defined in separate files in the same package
+  type extern = scala.scalanative.annotation.extern
```

Comment:
Why not declare `final class extern` right here? That would satisfy Scala 3's constraint without changing the source API at all. It might even preserve binary compat? (not sure)

Reply:
For some reason I thought that's not possible, probably I had some issues with that in 3.0.2 Scala version. Anyway, now it works just fine and we can define them both in a single file